### DISCUSSION TITLE
Next previous links with title

### DISCUSSION THREAD

**Discussion by keybasegit** [hbstack/.github#2] [Ideas]
Next and previous links under the post are step forward. Links with title might be better.

**Comment by razonyang** [comment]
It's worth to have an option for this, will do it on this weekend.

**Comment by razonyang** [comment]
Implemented by https://github.com/hbstack/docs/pull/58 and https://github.com/hbstack/blog/pull/70, thanks for sharing your ideas.

**Reply by keybasegit** [reply]
Solved. Thanks. @vlad
